package.json

```diff
@@ -51,9 +51,9 @@
     "expo-random": "~13.0.0",
     "expo-secure-store": "~12.0.0",
     "expo-sharing": "~11.0.1",
-    "expo-splash-screen": "~0.17.4",
+    "expo-splash-screen": "~0.17.5",
     "expo-status-bar": "~1.4.2",
-    "expo-updates": "~0.15.4",
+    "expo-updates": "~0.15.5",
```

Comment:
why did  you update package versions?

Reply:
Build was failing on CI and I thought it would be better to fix now than leave it to dependabot.

Reply:
oh interesting. it should have been caught before being merged to master, that's weird!